package.json

```diff
@@ -31,11 +32,11 @@
     "lint": "ng lint ccd-case-management-web",
     "e2e": "ng e2e",
     "start:fr": "ng serve --aot --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr",
-    "build:fr": "ng build --prod --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr",
+    "build:fr": "npm run config && ng build --prod --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr",
```

Comment:
rather than changing build:fr and build:client-and-server-bundles which both delegate to build, can't we change only `build`? I guess we want to call `npm run config` on every type of build?